client/src/panels/panel_declarations/finances/transfer_payments/tp_by_region.js

```diff
@@ -0,0 +1,108 @@
+import text from './tp_by_region.yaml';
+import {
+  formats,
+  run_template,
+  declare_panel,
+  years,
+  businessConstants,
+  create_text_maker_component,
+  StdPanel,
+  Col,
+  declarative_charts,
+} from "../../shared.js";
+import { Canada } from '../../../../charts/canada/index.js';
+
+const { std_years } = years;
+const formatter = formats["compact2_raw"];
+const includeNcr = false;
+
+const { text_maker } = create_text_maker_component(text);
+const { provinces } = businessConstants;
+const { A11YTable } = declarative_charts;
+
+const calculate_common = (years_by_province) => {
+  const max = d3.max(d3.values(_.last(years_by_province)));
+  const color_scale = d3.scaleLinear()
+    .domain([0,max])
+    .range([0.2,1]);
+  return {
+    years_by_province,
+    color_scale,
+    context_years: std_years,
+    formatter,
+    includeNcr,
+  };
+};
+
+const calculate_funcs_by_level = {
+  gov: function(){
+    const { orgTransferPaymentsRegion } = this.tables;
+    return calculate_common( std_years.map( year => orgTransferPaymentsRegion.prov_code(year, false) ) );
+  },
+  dept: function(subject){
+    const { orgTransferPaymentsRegion } = this.tables;
+    return calculate_common( std_years.map( year => orgTransferPaymentsRegion.prov_code(year, subject.unique_id) ) );
+  },
+};
+
+const prepare_data_for_a11y_table = (years_by_province) =>
+  _.chain(provinces)
+    .map((province, prov_code) => {
+      if(prov_code!="onlessncr" && prov_code!="qclessncr" && prov_code!="ncr"){
```

Comment:
This pattern can be cleaned up, IMO, with `_.includes`, e.g. `!_.includes(["onlessncr", "qclessncr", "ncr"], prov_code)`.

Side note, should be `!==`'s. Not specifically relevant here, but generally better practice (same as using `===` instead of `==`, the `==/!=` type coercion is usually not desired and frequently not what you'd expect).

Reply:
ah okay!! didn't actually know `!==` was a thing until now